### PR TITLE
Details SPF/DKIM on tutorial section

### PR DESCRIPTION
**Pull request by Mathieu-Lala**
# Summary of changes

* add vsmtp logo + badge to introduction
* split the "Hardening vSMTP" into 3 sections (Open Relay / SPF / DKIM)
* move the Email Authentication Mechanism chapter in the "Terminology"
* merge the tuto/dkim&eam/dkim and tuto/spf&eam/spf pages

### CODE REVIEW

File: src/tuto/0/dkim.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
-# DomainKeys Identified Message
+# Using DomainKeys Identified Mail
+
+The DKIM protocol is natively implemented in vSMTP.
+
+This protocol ensure that the content of the message has not been modified during the transport. A new DNS record is added into the `doe-family.com` DNS zone. It declares the public key usable to verify the messages.
+
+We will configure these rule:
```
Comment (ltabis) [inline src/tuto/0/dkim.md:7]: "these rules"

File: src/tuto/0/dkim.md [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+
+- The sender is a Doe's family account : a DKIM signature is added to the message.
```
Comment (ltabis) [inline src/tuto/0/dkim.md:9]: "The sender is an account from Doe's family."

File: src/tuto/0/dkim.md [hunk 3]
```diff
@@ -1,4 +1,42 @@
...
+- The recipient is a Doe's family account : the DKIM signatures are verified.
```
Comment (ltabis) [inline src/tuto/0/dkim.md:10]: Same as last comment.

File: src/tuto/0/open-relay.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Disabling [open mail relay](https://en.wikipedia.org/wiki/Open_mail_relay)
+
+The server must only accept messages from the Internet that comply with :
+
+- The recipient is a Doe's family account, whatever the sender is (incoming messages).
```
Comment (ltabis) [inline src/tuto/0/open-relay.md:5]: "The recipient is an account from Doe's family, whatever the sender is. (incoming messages)"

File: src/tuto/0/open-relay.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+- The sender is authenticated as a Doe's family account, whatever the recipient is (outgoing messages).
```
Comment (ltabis) [inline src/tuto/0/open-relay.md:6]: "The sender is authenticated as an account from Doe's family, whatever the recipient is. (outgoing messages)"

**Review by ltabis — CHANGES_REQUESTED** [review]

**Review by ltabis — APPROVED** [review]

File: src/tuto/0/spf.md [hunk 6]
```diff
@@ -1,12 +1,37 @@
...
+
+Edit the `/etc/vsmtp/rules/main.vsl` file and add the rule:
+
+```js
+// -- main.vsl
+#{
+  // ... previous code ...
+
+  mail: [
+    rule "check spf" || check_spf("both", "soft"),
```
Comment (paguardiolle) [inline src/tuto/0/spf.md:19]: Imho same as DKIM. Requires a "if not internal" then spf else nothing.

**Pull request merged**